src/app/pages/FeatureIdxPage/index.test.jsx

```diff
@@ -0,0 +1,219 @@
+});
+
+jest.mock('#containers/PageHandlers/withError', () => Component => {
+  return props => (
+    <div id="ErrorContainer">
+      <Component {...props} />
+    </div>
+  );
+});
+
+jest.mock('#containers/PageHandlers/withData', () => Component => {
+  return props => (
+    <div id="DataContainer">
+      <Component {...props} />
+    </div>
+  );
+});
+
+jest.mock('#containers/PageHandlers/withContexts', () => Component => {
+  return props => (
+    <div id="ContextsContainer">
+      <Component {...props} />
+    </div>
+  );
+});
+
+describe('Feature Idx Page', () => {
+  afterEach(() => {
+    fetchMock.restore();
+  });
+
+  describe('snapshots', () => {
+    it('should render an afrique feature idx page correctly', async () => {
+      fetchMock.mock(
+        'http://localhost/some-feature-idx-page-path.json',
+        JSON.stringify(afriqueFeatureIdxPageData),
+      );
+      const { pageData } = await getInitialData({
+        path: 'some-feature-idx-page-path',
+        service: 'afrique',
+      });
+
+      let container;
+      await act(async () => {
+        container = render(<FeatureIdxPageWithContext pageData={pageData} />)
+          .container;
+      });
+      expect(container).toMatchSnapshot();
+    });
+
+    it('should render an afrique amp feature idx page', async () => {
+      fetchMock.mock(
+        'http://localhost/some-feature-idx-path.json',
+        JSON.stringify(afriqueFeatureIdxPageData),
+      );
+      const { pageData } = await getInitialData({
+        path: 'some-feature-idx-page-path',
+        service: 'afrique',
+      });
+
+      let container;
+      await act(async () => {
+        container = render(
+          <FeatureIdxPageWithContext pageData={pageData} isAmp />,
+        ).container;
+      });
+      expect(container).toMatchSnapshot();
+    });
+  });
+
+  describe('Assertions', () => {
+    it('should render visually hidden text as h1', async () => {
+      fetchMock.mock(
+        'http://localhost/some-feature-idx-page-path.json',
+        JSON.stringify(afriqueFeatureIdxPageData),
+      );
+      const { pageData } = await getInitialData({
+        path: 'some-feature-idx-page-path',
+        service: 'afrique',
+      });
+
+      let container;
+      await act(async () => {
+        container = render(<FeatureIdxPageWithContext pageData={pageData} />)
+          .container;
+      });
+
+      const h1 = container.querySelector('h1');
+      const content = h1.getAttribute('id');
+      const tabIndex = h1.getAttribute('tabIndex');
+
+      expect(content).toEqual('content');
+      expect(tabIndex).toBe('-1');
+
+      // const span = h1.querySelector('span');
+      // expect(span.getAttribute('role')).toEqual('text');
+      expect(h1.textContent).toEqual('Tout savoir sur la CAN 2019');
+
+      // const langSpan = span.querySelector('span');
+      // expect(langSpan.getAttribute('lang')).toEqual('en-GB');
+      // expect(langSpan.textContent).toEqual('BBC News');
```

Comment:
We still need to figure out what to do about this - it's probably an a11y question